caesar_rest/job_configurator.py

```diff
@@ -533,11 +533,10 @@ def transform_inputfile(self,file_uuid):
 		use_mongo= (mongo_enabled and has_mongo)
 
 		# - Get aai info
-		aai_enabled= current_app.config['USE_AAI']
-		has_oidc= (oidc is not None)
 		username= 'anonymous'
-		if aai_enabled and has_oidc:
-			username= oidc.user_getfield('preferred_username')
+		if ('oidc_token_info' in g) and (g.oidc_token_info is not None and 'email' in g.oidc_token_info):
+			username=g.oidc_token_info['email']
+		logger.warn(username)
```

Comment:
I added these warns for quickly seeing the username, but of course we can remove them